crates/wasmtime/src/config.rs

```diff
@@ -1454,3 +1462,47 @@ pub enum WasmBacktraceDetails {
     /// `WASMTIME_BACKTRACE_DETAILS` environment variable.
     Environment,
 }
+
+/// Select which WASI modules are available at runtime for use by Wasm programs.
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub struct WasiModules {
```

Comment:
I don't think this needs to be in the `wasmtime` crate since I don't think it's read by anything in the `wasmtime` crate?

Reply:
Yup, moved to `src/lib.rs`.